config/routes.rb

```diff
@@ -6,5 +6,11 @@
     end
   end
 
+  namespace :api, defaults: { format: 'json' } do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.